newrelic_lambda_cli/cli/subscriptions.py

```diff
@@ -0,0 +1,68 @@
+import json
+
+import boto3
+import click
+
+from .. import integrations, permissions
+from .cliutils import done
+from .decorators import add_options, AWS_OPTIONS
+
+
+@click.group(name="subscriptions")
+def subscriptions_group():
+    """Manage New Relic AWS Lambda Layers"""
```

Comment:
Update docstring. This is what is displayed next to the group for `newrelic-lambda --help`.